packages/admin-ui/src/pages/rollups/components/Zkevm.tsx

```diff
@@ -0,0 +1,126 @@
+import Loading from "components/Loading";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+import SubTitle from "components/SubTitle";
+import "./columns.scss";
+import { Alert, Button } from "react-bootstrap";
+import ZkevmCommunity from "./ZKevmCommunity";
+import ZkEvm from "./columns/ZkEvm";
+import { responseInterface } from "swr";
+import { useZkEvmConfig } from "./useZkEvmConfig";
+
+type ZkEvmConfigGetResponse = responseInterface<ZKEVMItem<"rollup">[], Error>;
+
+const zkEvmConfigGet = (): ZkEvmConfigGetResponse => {
+ 
+  return {
+    data: [], 
+    revalidate: async () => true,
+    mutate: async () => [],
+    isValidating: false
+  };
+};
+
+export default function Zkevm({ description }: { description: string }) {
+  const { theme } = React.useContext(AppContext);
+  const currentZkEvmConfigReq = zkEvmConfigGet();
+
+  // hooks
+  const { reqStatus, isZkEvmInstalled, setReqStatus } = useZkEvmConfig(
+    currentZkEvmConfigReq
+  );
+
+  /**
+   * Set new zkEVM config
+   */
+  async function setNewZkEvmConfig() {
+    try {
+      // logic for setting the zkEVM configuration here
+    } catch (e) {
+      setReqStatus({ error: e });
+    } finally {
+      // logic for updating the zkEVM configuration here
+    }
+  }
+
+  function setNewZkEvmItem(value: any): void {
+    throw new Error("Function not implemented.");
+  }
+
+  return (
+    <div className={theme === "light" ? "optimism-light" : "optimism-dark"}>
+      {/* Render the ZkEvmCommunity component */}
+      <ZkevmCommunity />
+      {currentZkEvmConfigReq.data ? (
+        <Card>
+          {/* Render the description */}
+          <p>{description}</p>
+
+          <Row className="staker-network">
+            <Col>
+              <SubTitle>zkEVM Modules</SubTitle>
+              {/* Map over the zkEVM items */}
+              {currentZkEvmConfigReq.data.map(
+                (zkEvmItem: ZKEVMItem<"rollup">, i: number) => (
+                  <ZkEvm
+                    key={i}
+                    zkEvmItem={zkEvmItem}
+                    setNewZkEvmItem={setNewZkEvmItem}
+                    isSelected={false} 
+                  />
+                )
+              )}
+            </Col>
+
+            <Col></Col>
+
+            <Col></Col>
+          </Row>
+
+          <hr />
+
+          <div>
+            <div className="staker-buttons">
+              {/* Button to apply changes */}
+              <Button
+                variant="dappnode"
+                disabled={!isZkEvmInstalled || reqStatus.loading}
+                onClick={() => setNewZkEvmConfig()}
+              >
+                Apply changes
+              </Button>
+            </div>
+
+            {/* Render alert if changes are not allowed */}
+            {!isZkEvmInstalled && (
+              <>
+                <br />
+                <br />
+                <Alert variant="danger">
+                  Cannot apply changes: zkEVM is not installed
```

Comment:
When we cant apply changes, normally it is because no changes are detected.

I think its best to disable it by default and use booleans to define if it is enabled. See
https://github.com/dappnode/DNP_DAPPMANAGER/blob/gi/rollup-zkevm/packages/admin-ui/src/pages/stakers/components/StakerNetwork.tsx#L265